crud/src/main/java/com/redhat/lightblue/rest/crud/health/CrudCheckConfiguration.java

```diff
@@ -87,12 +90,12 @@ private RolesProvider createRolesProvider() throws Exception {
 
         LdapConfiguration ldapConfiguration = new LdapConfiguration();
         ldapConfiguration.initializeFromJson(root);
-        
+
         return new CachedRolesProvider(new LdapRolesProvider(searchBase, ldapConfiguration),
```

Comment:
Ah, I guess we can't get this from the `CertLdapLoginModule.getRolesProvider()` because that's in a separate class loader? Have we tried making that method initialize the provider if it isn't initialized yet? Perhaps even if we can't share the same role provider instance, we can still reuse the initialization code.

/me curses app servers (yet again)